### PR TITLE
chore(*): update github template

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Update github template

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @codeflitting @FogDong 

**Release note**:

```release-note
update github template
```

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
+/kind regression
+/kind upgrade
+/kind denpendency
+
+Also with one of following areas:
+/area auth
+/area resource
+/area apps
+/area cicd
+/area cargo
```
Comment (codeflitting) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: 把 Klever 相关的保留下就好了？

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,8 +1,40 @@
...
+/area insight
+/area loadbalancer
```
Comment (codeflitting) [inline .github/PULL_REQUEST_TEMPLATE.md:25]: Get rid of useless labels?

Comment (simon-cj) [reply .github/PULL_REQUEST_TEMPLATE.md]: comments are addressed, PTAL

**Comment by simon-cj** [comment]
ping @FogDong

**Review by FogDong — COMMENTED** [review]
/lgtm

**Pull request merged**